.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  ignorePatterns: ["out", "/*.js"],
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint'],
+  root: true,
+
+};
```

Comment:
Bonus - I added an default typescript-eslint config file. Didn't want to submit any dumb code!